doc/man3/SSL_CTX_sess_set_get_cb.pod

```diff
@@ -98,6 +98,16 @@ If the get_session_cb() does not write to B<copy>, the reference count
 is incremented and the session must be explicitly freed with
 L<SSL_SESSION_free(3)>.
 
+The get_session_cb() is called on SSL/TLS clients with the application-defined
+cache identifier set by the client. The get_session_cb() is always called when
+session caching is disabled. The get_session_cb() is passed the
+B<ssl> connection, the cache identifier of length B<length> at the memory location
+B<data>. With the parameter B<copy> the callback can require the
```

Comment:
The paragraph immediately above this refers to the `get_session_cb()` when acting as a server, and it also references `length` and `data`. So that one is bad, too. None of the callbacks are defined separately, but I added the arguments to the `SSL_CTX_set_get_cb()` function.